internal/util.go

```diff
@@ -12,13 +12,17 @@ import (
 // for resources that can contain an arbitrary or dynamic number of fields.
 func RemainingKeys(s interface{}, m map[string]interface{}) (extras map[string]interface{}) {
 	extras = make(map[string]interface{})
+	for k, v := range m {
+		extras[k] = v
+	}
+
 	valueOf := reflect.ValueOf(s)
 	typeOf := reflect.TypeOf(s)
 	for i := 0; i < valueOf.NumField(); i++ {
 		field := typeOf.Field(i)
 		tagValue := field.Tag.Get("json")
-		if _, ok := m[tagValue]; !ok {
-			extras[tagValue] = m[tagValue]
+		if _, ok := extras[tagValue]; ok {
+			delete(extras, tagValue)
```

Comment:
`IsAdmin` wouldn't be ignored, but it would go in `Extra` because:
1. it has no struct tag, **and**
2. there is no `isAdmin` field in the JSON response.